gateway-js/src/datasources/RemoteGraphQLDataSource.ts

```diff
@@ -252,16 +244,16 @@ export class RemoteGraphQLDataSource<
       // thus the overall response) is uncacheable. (If you don't like this, you
       // can tweak the `cache-control` header in your `didReceiveResponse`
       // method.)
-      const hint: CacheHint = { maxAge: 0 };
+      const hint: GatewayCacheHint = { maxAge: 0 };
       const maxAge = parsed['max-age'];
       if (typeof maxAge === 'string' && maxAge.match(/^[0-9]+$/)) {
         hint.maxAge = +maxAge;
       }
       if (parsed['private'] === true) {
-        hint.scope = CacheScope.Private;
+        hint.scope = 'PRIVATE';
```

Comment:
This change has a runtime effect, but a trivial one as `CacheScope.Private`'s runtime value is `'PRIVATE'`.